### PR TITLE
osm2pgsql 0.87.0

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/osm2pgsql.rb [hunk 1]
```diff
@@ -16,23 +14,42 @@ class Osm2pgsql < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "boost" => :build
```
Comment (mikemcquaid) [inline Library/Formula/osm2pgsql.rb]: @bfontaine Just FYI this is a runtime dependency too.

Comment (bfontaine) [inline Library/Formula/osm2pgsql.rb]: @mikemcquaid Yeah thanks, I saw your commit. How did you see that?

Comment (mikemcquaid) [inline Library/Formula/osm2pgsql.rb]: http://bot.brew.sh/job/Homebrew/11482/ failed which posts to https://groups.google.com/forum/#!topic/brew-test-bot/p_JPWHoLmSQ

Comment (mikemcquaid) [inline Library/Formula/osm2pgsql.rb]: (it tried to install the bottle and run the test which failed)

Comment (bfontaine) [inline Library/Formula/osm2pgsql.rb]: Thanks, I’ll try to check that in the future.

**Pull request closed without merging**